openlibrary/templates/covers/change.html

```diff
@@ -29,7 +29,7 @@
 
 <div class="$size sansserif manageCoversContainer hidden--nojs">
   <a aria-controls="addImage-$imagesId" class="coverPop dialog--open">
-    <div class="manageCovers" data-config="$dumps({'key': doc.type.key, 'url': get_coverstore_public_url(), 'selector': cover_selector, 'add_url': add_url, 'manage_url': manage_url})">$change</div>
+    <div class="manageCovers hidden" data-config="$dumps({'key': doc.type.key, 'url': get_coverstore_public_url(), 'selector': cover_selector, 'add_url': add_url, 'manage_url': manage_url})">$change</div>
```

Comment:
Which is that we should inline the positioning CSS here to avoid the loading flash 😁 Do leave a comment though so that future folks understand why it's added!

```suggestion
    <div class="manageCovers" data-config="$dumps({'key': doc.type.key, 'url': get_coverstore_public_url(), 'selector': cover_selector, 'add_url': add_url, 'manage_url': manage_url})">$change</div>
```